### PR TITLE
add solid beets vault

### PR DESCRIPTION
**Pull request by andreb0x**
want: 0xf2b2103e1351c4d47b27efc8100bb264364fa92c
chefPoolId: 59
vault: 0xf34bED1111dA686B2AB336b7ec594A22Df6b87d7
strat: 0x453256366F7A3F58b9e07887Be329484034168d2

### CODE REVIEW

File: src/features/configure/vault/fantom_pools.js [hunk 1]
```diff
@@ -140,7 +140,33 @@ export const fantomPools = [
...
+    oracle: 'lps',
+    oracleId: 'beets-solid-beets',
+    oraclePrice: 0,
+    depositsPaused: false,
+    status: 'active',
+    platform: 'Beethoven X',
+    assets: ['SOLID', 'BEETS'],
+    risks: ['COMPLEXITY_LOW', 'NEW_STRAT', 'IL_HIGH', 'MCAP_SMALL', 'AUDIT', 'CONTRACTS_VERIFIED'],
+    stratType: 'StratLP',
+    withdrawalFee: '.01%',
```
Comment (roman-monk) [inline src/features/configure/vault/fantom_pools.js:165]: why in this format ".01%"?

**Comment by roman-monk** [comment]
What is `MyBal` contract (file 17 of 17)? That's not the part of our codebase

**Comment by roman-monk** [comment]
the vault is fine, but the strat deployment seems to be broken, lots of unused contracts included there

**Comment by andreb0x** [comment]
> 

oh shit... that's embarrassing. I was just doodling for my own learning sake. i didnt mean to deploy that.

**Comment by andreb0x** [comment]
I can redploy strat, pointing to existing vault. but then can someone point vault to new strat?

**Comment by andreb0x** [comment]
just realizing the vaultOwner and stratOwner addresses I used in the solid beets strat are from the old addressbook. https://github.com/beefyfinance/address-book/blob/main/address-book/fantom/platforms/beefyfinance.ts

I'm closing this PR, will redeploy new vault and strat.

**Pull request closed without merging**